src/python/torch_radon/radon.py

```diff
@@ -169,14 +169,14 @@ def filter_sinogram(
         pad = padded_size - size
         padded_sinogram = torch.nn.functional.pad(sinogram.float(), (0, pad, 0, 0))
 
-        sino_fft = torch.fft.rfft2(padded_sinogram) / np.sqrt(padded_size)
+        sino_fft = torch.fft.rfft(padded_sinogram, norm="ortho")
```

Comment:
The FFT-related code changes LGTM. In my testing of filtered backprojection (of 2 sinograms), the changed code returned no significant difference in numerical value from the previous version, but the `requires_grad` did remain `True`. My testing script is like:
```Python
# In the following, `myradon_tool` and `myradon_tool3` are instances of the Radon class (with projection and FBP methods). The only difference in that FFT-operations in `myradon_tool` are built on `cuda_backend`, while those in `myradon_tool3` are all replaced by `torch.fft`.

sino = torch.FloatTensor(gt_sino).unsqueeze(0).unsqueeze(0).cuda()
sino.requires_grad = True

img_old_1 = myradon_tool.fan_fbp_torch(sino, angles)
sino_old_1 = myradon_tool.fan_proj_torch(img_old_1, angles).clip(sino.min(), sino.max())
img_old_2 = myradon_tool.fan_fbp_torch(sino_old_1, angles)

img_new_1 = myradon_tool3.fan_fbp_torch(sino, angles)
sino_new_1 = myradon_tool3.fan_proj_torch(img_new_1, angles).clip(sino.min(), sino.max())
img_new_2 = myradon_tool3.fan_fbp_torch(sino_new_1, angles)

print('>>>>>> default filtering')
print('requires_grad?: ', img_old_1.requires_grad, sino_old_1.requires_grad, img_old_2.requires_grad)
print(torch.abs(img_old_2-img_old_1).mean(), torch.abs(sino-sino_old_1).mean())
print('>>>>>> differentiable filtering')
print('requires_grad?: ', img_new_1.requires_grad, sino_new_1.requires_grad, img_new_2.requires_grad)
print(torch.abs(img_new_2-img_new_1).mean(), torch.abs(sino-sino_new_1).mean())
```
And the results are like:
```
>>>>>> default filtering
requires_grad?:  False True False
tensor(0.0026, device='cuda:0') tensor(0.0433, device='cuda:0', grad_fn=<MeanBackward0>)
>>>>>> differentiable filtering
requires_grad?:  True True True
tensor(0.0026, device='cuda:0', grad_fn=<MeanBackward0>) tensor(0.0433, device='cuda:0', grad_fn=<MeanBackward0>)
```